submissions/KostiantynOkhorzin/frogger-game/app.js

```diff
@@ -0,0 +1,87 @@
+class Enemy {
+    constructor(x, y, speed, sprite = 'images/enemy-bug.png') {
+        this.x = x;
+        this.y = y;
+        this.speed = speed;
+        this.sprite = sprite;
+    }
+
+    update(dt) {
+        this.x += this.speed * dt;
+
+        if (this.x > 510) {
+            this.x = -50;
+            this.speed = Math.round(Math.random() * 100 + 100); 
+        };
+
+        if (player.x < this.x + 80 && player.x + 80 > this.x &&
+            player.y < this.y + 60 && 60 + player.y > this.y) {
+            player.x = 200;
+            player.y = 400;
```

Comment:
As a peer developer that needs to contribute to this project I want to understand semantics behind all these magic numbers here and below in the code.

Also apparently `update` method doesn't just updates position but also has some more of business logic. We need to separate concerns. Please move checks to a different method.